content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -4,19 +4,22 @@ This article is about Sockets. It will give you a glimpse of how clients and ser
 
 Socket programming is a means of communicating data between two computers across a network. Connections can be made using either a **connection-oriented protocol** or a **connectionless protocol**. TCP/IP, a connection-oriented protocol, will be used in our case.
 
-Before exchanging data, two computers must establish a link in connection-oriented protocols. In contrast, you don't require every packet to arrive at its intended destination for connectionless. It is made feasible via UDP (User Datagram Protocol). To demonstrate sockets further, we shall use the Client\Server architecture. Client and server communicate by writing to and reading from the socket connection. The machines must have each other's network information to make this connection.
+Before exchanging data, two computers must establish a link in connection-oriented protocols. In contrast, you don't require every packet to arrive at its intended destination for connectionless. It is made feasible via UDP (User Datagram Protocol).
 
-Table of contents:
-- [Definition-of-a-Socket](#definition-of-a-socket)
-- [Programming-on-the-Server-side ](#programming-on-the-server-side)
-- [Programming-on-the-Client-side](#programming-on-client-side)
-- [How-to-Run-Programs-in-a-Variety-of-Applications](#how-to-run-programs-in-a-variety-of-applications)
+To demonstrate sockets further, we shall use the Client/Server architecture. Client and server communicate by writing to and reading from the socket connection. The machines must have each other's network information to make this connection.
+
+### Table of contents:
+- [Definition of a Socket](#definition-of-a-socket)
+- [Programming on the Server-side ](#programming-on-the-server-side)
+- [Programming on the Client-side](#programming-on-client-side)
+- [How to Run Programs in a Variety of Applications](#how-to-run-programs-in-a-variety-of-applications)
 - [Conclusion](#conclusion)
 
 ### Definition of a Socket
 
 A Socket is a communication **endpoint** that serves as a link between two machines on a network. It has a socket number, which the TCP/IP layer can use to identify the application that receives the data. Additionally, a port number and an IP address are what make up an endpoint.
 
+#### What is a TCP?
 Transmission Control Protocol (TCP) - This is one of the most important protocols of the Internet Protocols suite. It is a widely used protocol for data transmission in a network that operates in Client/Server point-to-point mode.
 
 Two categories of Sockets:
```

Comment:
Maybe add another header before this because it looks like this is under the \#\#\# What is a TCP? section